.circleci/config.yml

```diff
@@ -13,58 +33,59 @@ commands:
           <<: *cache-key
           paths:
             - ~/.cache/yarn
+  yarn_serve:
+    steps:
+      - run:
+          command: yarn serve .docz/dist -n -s -p 9000
+          background: true
+      - run: yarn wait-on -t 60000 http://localhost:9000
 
 jobs:
   lint:
-    docker:
-      - image: cypress/browsers:node14.17.0-chrome88-ff89
-    environment:
-      CYPRESS_INSTALL_BINARY: 0
+    executor: linux-node
     steps:
       - checkout
       - yarn_install
       - run: yarn lint
       - run: yarn build
       - run: yarn tsc
   unit-tests:
-    docker:
-      - image: cypress/browsers:node14.17.0-chrome88-ff89
-    environment:
-      CYPRESS_INSTALL_BINARY: 0
+    executor: linux-node
     steps:
       - checkout
       - yarn_install
       - run: yarn build
       - run: yarn test:ci
+
   integration-tests:
-    resource_class: medium+
+    parameters:
+      executor:
+        type: executor
+      browser:
+        type: enum
+        enum: [chrome, firefox, edge]
```

Comment:
It seems that `edge` for linux is still in Beta. Is this a problem? Can we rely on it?

Reply:
I will just keep it optional for now.